src/main/java/com/qunite/api/service/QueueServiceImpl.java

```diff
@@ -24,7 +25,10 @@ public class QueueServiceImpl implements QueueService {
 
   @Override
   @Transactional
-  public Queue create(Queue queue) {
+  public Queue create(Queue queue, String username) {
+    queue.setCreator(userService.findByUsername(username)
+        .orElseThrow(() -> new UserNotFoundException(
+            "Invalid creator with username: %s".formatted(username))));
```

Comment:
Why did you write comprehensive error message here and returned only username in UserDetailsServiceImpl.loadUserByUsername?
Maybe define the kind of 'standart' for error messages?

Reply:
Or maybe use different exception class here